src/osal/lv_freertos.h

```diff
@@ -51,7 +51,7 @@ extern "C" {
 
 typedef struct {
     void (*pvStartRoutine)(void *);       /**< Application thread function. */
-    void * xTaskArg;                      /**< Arguments for application thread function. */
+    void * pTaskArg;                      /**< Arguments for application thread function. */
```

Comment:
`pvTaskArg`

Reply:
In the coding standard naming conventions you linked, the "v" seems to only apply for function names that return `void` (nothing), not void pointer variables.